cypress/e2e/reservations.cy.js

```diff
@@ -0,0 +1,32 @@
+import * as page from '../fixtures/URLs.json';
+import ProfilePage from '../support/page_objects/profilePage';
+
+describe('Reserve and Unreserve item for a user test suite', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('donorLoginEmail'), Cypress.env('loginPassword'));
+  });
+
+  it('user can mark an item as reserved for a user that has requested it', () => {
+    cy.visit(page.profile, { failOnStatusCode: false });
+    ProfilePage.reserveButton('shoes').click();
+    cy.get('button').contains('Confirm').should('be.visible').click();
+    cy.get('.card')
+      .contains('shoes')
+      .parents('li')
+      .find('button')
+      .contains('Unreserve')
+      .should('be.visible');
+  });
+
+  it('user can remove the reservation status of their item', () => {
+    cy.visit(page.profile, { failOnStatusCode: false });
+    const unreserveButton = cy
+      .get('.card')
+      .contains('shoes')
+      .parents('li')
+      .find('button')
+      .contains('Unreserve')
+      .should('be.visible');
+    unreserveButton.click();
+  });
```

Comment:
```suggestion
  it('user can mark an item as reserved for a user that has requested it', () => {
    cy.visit(page.profile, { failOnStatusCode: false });
    ProfilePage.reserveButton('shoes').click();
    cy.get('button').contains('Confirm').should('be.visible').click();
    cy.get('.card')
      .contains('shoes')
      .parents('li')
      .find('button')
      .contains('Unreserve')
      .should('be.visible');
      cy.get('button').contains('Confirm').should('be.visible').click();
  });

  it('user can remove the reservation status of their item', () => {
    cy.visit(page.profile, { failOnStatusCode: false });
    cy.get('button').contains('Confirm').should('be.visible').click();
    const unreserveButton = cy
      .get('.card')
      .contains('shoes')
      .parents('li')
      .find('button')
      .contains('Unreserve')
      .should('be.visible');
    unreserveButton.click();
  });
```

I tried adding this one check to the tests and it highlights one adjustment that needs to be made now that you've updated the database property `is_reserved`. We're still evaluating for `item.reserved` when rendering the cards so we end up loosing the "Reserved" tag on the UI.

With this one extra assertion we can make sure to have that back once the test is passing.